tests/test-ajax.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * Tests for the back-end handling of sticky meta data.
+ *
+ * @package Sticky_Tax
+ */
+
+use LiquidWeb\StickyTax\Ajax as Ajax;
+
+/**
+ * Test suite handling the post meta related to sticky taxonomy posts.
+ */
+class AjaxTest extends WP_Ajax_UnitTestCase {
+
+	public function test_ajax_success() {
+		$_POST['nonce']     = wp_create_nonce( 'sticky-tax' );
+		$_POST['action']    = 'stickytax_get_id_from_name';
+		$_POST['term_name'] = 'Uncategorized';
+		$_POST['term_type'] = 'category';
+
+		try {
+			$this->_handleAjax( 'stickytax_get_id_from_name' );
+		} catch ( WPAjaxDieContinueException $e ) {
+			return;
```

Comment:
Returning early from a test will consider it to have passed (since no assertions failed), which is likely not the intended result. Instead, I'd recommend dropping the `try/catch` block (thus letting any errors surface to the test runner) or explicitly failing the test if something goes wrong:

```php
try {
  // Something that might fail.

} catch ( MyException $e ) {
  $this->fail( 'Caught unexpected exception: $e->getMessage() );
}
```

Reply:
I initially had a try/catch in there, but that's now how the Ajax unit tests are supposed to run based on all the documentation I found

Reply:
to clarify: if you can get that working as expected with a try / catch, I'm all for it.

Reply:
to clarify: I tried a few different ways of structuring that test, this is the only one that seemed to work correctly.

Reply:
It turns out you're right — because [`wp_send_json()` (and its related functions) use `wp_die()` when handling ajax requests](https://developer.wordpress.org/reference/functions/wp_send_json/), our tests have to explicitly catch the exceptions. Which is...well, one of the many reasons this isn't technically *unit* testing.

Instead, we want to use a `try/catch` but not return from within the `catch` portion, as that would bypass our test assertions, potentially leading to false positives. I've taken care of this in 83ed2c6e2.